lib/moyasar/actions/refund.rb

```diff
@@ -0,0 +1,23 @@
+module Moyasar
+  module Actions
+    module Refund
+
+      def self.included(klass)
+        klass.extend         ClassMethods
+        klass.send :include, InstanceMethods
+      end
+
+      module ClassMethods
+        def refund(id, attrs = {})
+          response = request(:post, refund_url(id), params: attrs)
+
+          new(response.body)
+        end
+      end
+
+      module InstanceMethods
```

Comment:
Tests are not aligned correctly with not refund, 
but the other resources like update and create.

The current for refund like them. Will work on tests next ..